run.sh

```diff
@@ -1,5 +1,4 @@
 forge test --match-contract UnitTest -vvv 
-forge test --match-contract IntegrationTest --fork-url ethereum -vvv
```

Comment:
I think this was supposed to run ChainlinkCompositeOracleBase, so it should be put back and matches `IntegrationBaseTest` instead

Reply:
I made a mistake when I renamed `ChainlinkCompositeOracleIntegrationTest` to `ChainlinkCompositeOracleIntegrationTestBase`. My intention was to migrate the test to base, but I couldn't because Chainlink doesn't have all the oracles the test uses on Base. So, I think we should rename it back to `ChainlinkCompositeOracleIntegrationTest` and add the test to CI again.